pkgcfg/package-configuration/apacheHttpd.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "apache",
+  "version": "0.0.1",
+  "readme": "To start use \"apachectl start -f $HTTPD_CONFDIR/httpd.conf\".",
+  "env": {
+    "HTTPD_CONFDIR": "{{ .DevboxRoot }}/conf/apache"
+  },
+  "create_files": {
+    ".devbox/conf/apache/httpd.conf": "apache/httpd.conf"
```

Comment:
Similar to nginx, I feel like these default files can be directly copied from the `/nix/store`, instead of copying them from this repo. It seems like we need a way to specify files that we need to copy over from the immutable nix store to `.devbox/`

Almost like
```
".devbox/conf/apache/httpd.conf": "{{ .packagePath }}/lib/httpd.conf"
```

Reply:
@LucilleH that's reasonable (@loreto @Lagoja and I chatted about it a bit). From what I can tell though the default apache config is not 100% ready to use. So while it makes sense to try to use it as a base, it would require some editing if we want it to be ready to use without changes.

We chatted about potentially bringing over all of the nix store files (some as symlinks and others as simple rewrites so their internal paths point to right place) but we're punting on it for now because of complexity.

Anyway, since these are versioned, we can always improve them without being afraid of breaking people's setup.